source/winVersion.py

```diff
@@ -124,20 +183,23 @@ def isUwpOcrAvailable():
 	return os.path.isdir(UWP_OCR_DATA_PATH)
 
 
-WIN10_RELEASE_NAME_TO_BUILDS = {
-	"1507": 10240,
-	"1511": 10586,
-	"1607": 14393,
-	"1703": 15063,
-	"1709": 16299,
-	"1803": 17134,
-	"1809": 17763,
-	"1903": 18362,
-	"1909": 18363,
-	"2004": 19041,
-	"20H2": 19042,
-	"21H1": 19043,
-}
+# Deprecated: Windows 10 releases will be obtained from Windows Registry, no entries will be added.
+# The below map will be removed in 2022.1.
+if version_year < 2022:
+	WIN10_RELEASE_NAME_TO_BUILDS = {

```

Comment:
Please add a change log entry to the PR description explaining that this map is deprecated.